### ISSUE TITLE
Foxbit - CreateLimitBuyOrder  - Incorrect Side tag 

### ISSUE THREAD

**Issue by leogariba** [ccxt/ccxt#638]
Hi, Kroitor, sorry for my previous (and incomplete) post. 

In this moment I'm trying to use the foxbit instance. When I try to create a **limit buy order** I get an Status Error 500 as described below:

Code snippet:
 console.log (foxbit.id, await foxbit.createLimitBuyOrder ('BTC/BRL', 100000000, 1000000000000));

Error:
"(node:11672) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: foxbit {"Status":500,"Description":"**Invalid value tag(Side)=Buy**","Result":{}}"

This is erros is throw 'cos Side tag should be "1" or "2", not "Buy" or "Sell" as you can see here:
https://blinktrade.com/docs/#send-order

I was able to place my order changing the createOrder() function in foxbit.js as shown below:

let order = {
            'ClOrdID': this.nonce (),
            'Symbol': market['id'],
            **'Side': side === "buy" ? "1":"2",**
            'OrdType': '2',
            'Price': price,
            'OrderQty': amount,
            'BrokerID': market['brokerId'],
        };

Other exchanges that extend foxbit should face the same problem.

API Version : "ccxt": "1.10.176"
API Endpoint: createLimitBuyOrder

**Comment by kroitor** [comment]
Hi! Thanks for the description, we will fix it shortly. Please don't duplicate issues, only open one issue per problem. You can open a previously closed issue if you think it wasn't resolved. But for now, let's continue here. I'll update you as soon as this is fixed. Thx!

**Comment by kroitor** [comment]
This is fixed in ccxt 1.10.179+. Don't forget to update to most current version, because we are constantly uploading multiple fixes and improvements. We will be happy if you report back if it now works normally for you or not. Let us know if you have any difficulties or questions. Thx!

P.S. Aside of the issue, a short hint for the future: when you paste code or console output, make it readable. Developers will appreciate that. Work on formatting the question properly, that makes 50% of success. Read and follow recommendations on how to open issues (most projects have them). Most of the time you can even solve an issue yourself if you just follow those recommendations precisely.

- https://github.com/ccxt/ccxt/wiki#troubleshooting
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

I know, most users don't read docs, but if you start doing that and propagate the same behaviour all around, you'd get answers and help from other developers much quicker, believe me ;) 

Most websites allow code formatting including the GitHub as well. So, when pasting, enclose the code with triple backticks (&#96;&#96;&#96;), like shown below:

&#96;&#96;&#96;
your code or verbose console output
&#96;&#96;&#96;

↓ will result in ↓

```
let order = {
            'ClOrdID': this.nonce (),
            'Symbol': market['id'],
            **'Side': side === "buy" ? "1":"2",**
            'OrdType': '2',
            'Price': price,
            'OrderQty': amount,
            'BrokerID': market['brokerId'],
        };
``` 

↑ That's much better and easier to read, right? ) Thx again for your feedback ;)

**Comment by leogariba** [comment]
Thank you Kroitor! Sorry for the duplicate issue and unformatted code. Still learning the "github way". Just forked the repo to evaluate if I am capable to help you. I am from Brazil and there are others exchanges to work on.
And hell yeah! That's much better and easier to read! ;)
